### ISSUE TITLE
Can the input four views of the gaussian unet be four random viewpoints and how to input the pose?

### ISSUE THREAD

**Issue by cwchenwang** [3DTopia/LGM#4]

**Comment by ashawkey** [comment]
Yes, actually the model is trained with 4 random views. You could check this https://github.com/3DTopia/LGM/blob/main/core/models.py#L61 to specify the camera poses.

**Comment by cwchenwang** [comment]
Thank you for your helpful answer! It would be even better if we can pass the pose as a parameter.

**Comment by raushanagrawal** [comment]
one object have 4 view up, down, right or left how to input for this model for create one 3d model.

**Comment by ch1998** [comment]
> Yes, actually the model is trained with 4 random views. You could check this https://github.com/3DTopia/LGM/blob/main/core/models.py#L61 to specify the camera poses.

I used the specified camera pose for 3D generation and used the lego data of nerf_synthesis, but the synthesis effect was not ideal.
![image](https://github.com/3DTopia/LGM/assets/37291486/12b7ea57-6ee1-4813-a720-e70f805d9787)

result：
![image](https://github.com/3DTopia/LGM/assets/37291486/e0eb7191-cfbe-4d8d-a655-337898c09f5b)

code of pose：
![image](https://github.com/3DTopia/LGM/assets/37291486/25015afd-7bfb-4b81-928a-13d039e7cf41)

Does the camera pose require some transformation?

**Comment by ashawkey** [comment]
@cwchenwang You need to make sure the pose is in OpenGL world system & camera convention (https://kit.kiui.moe/camera/). I remember the poses directly from their json is in blender world system.

**Comment by cwchenwang** [comment]
Thank you! I found by converting elevation and azimuth to poses are much easier to do.
